cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt

```diff
@@ -45,6 +45,7 @@ import org.ossreviewtoolkit.clearlydefined.ClearlyDefinedService.Patch
 import org.ossreviewtoolkit.clearlydefined.ClearlyDefinedService.Server
 import org.ossreviewtoolkit.clearlydefined.ContributionType
 import org.ossreviewtoolkit.clearlydefined.ErrorResponse
+import org.ossreviewtoolkit.clearlydefined.HARVEST_CREATED
```

Comment:
Maybe add a few commas to the commit message for readability:
> Request harvesting of non-harvested components, for which curations
> should be uploaded, in order to allow uploading curations for them at a
> later time, when harvesting has finished, by simply running the command
> again.

Reply:
I believe English has different comma setting rules than German and have instead split the sentence in two.